src/open_company_web/core.cljs

```diff
@@ -10,6 +11,7 @@
             [open-company-web.components.login :refer (login)]
             [open-company-web.lib.raven :refer (raven-setup)]
             [open-company-web.lib.utils :as utils]
+            [open-company-web.actions]
```

Comment:
The `open-company-web.actions` namespace extends a multimethod defined in `open-company-web.dispatcher` if we don't require it it will not be part of the build and thus all dispatched actions won't have an appropriate handler.